addons/purchase/models/purchase.py

```diff
@@ -613,6 +626,83 @@ def retrieve_dashboard(self):
 
         return result
 
+    def _send_reminder_mail(self):
+        template = self.env.ref('purchase.email_template_edi_purchase_reminder', raise_if_not_found=False)
+        if template:
+            mail_id = template.with_context(is_reminder=True).send_mail(self.id, raise_exception=False, notif_layout="mail.mail_notification_paynow")
```

Comment:
as _send_reminder_mail isn't overridden anywhere, you could inline it in send_reminder_mail